sdk/storage/azure-storage-files-shares/inc/azure/storage/files/shares/share_options.hpp

```diff
@@ -620,7 +620,7 @@ namespace Azure { namespace Storage { namespace Files { namespace Shares {
      * hash of the content that has arrived with the header value that was sent. If the two hashes
      * do not match, the operation will fail with error code 400 (Bad Request).
      */
-    Azure::Core::Nullable<std::string> TransactionalMd5;
+    Azure::Core::Nullable<ContentHash> TransactionalMd5;
```

Comment:
What if in the future crc64 is supported for this API? We'll have to make a breaking change.